static/includes/apps/AppsDatasets.md

```diff
@@ -0,0 +1,63 @@
+
+Back up and restore functionality as well as migration of the ix-apps dataset from one apps pool to another are not currently supported by the TrueNAS interface.
+Support for these features is planned for a future TrueNAS release version.
+
+{{< hint type="note" title="ix-Applications Dataset" >}}
+TrueNAS 24.04 and earlier versions stored applications data in an **ix-applications** dataset on the configured apps pool.
+Systems with applications deployed that upgrade from earlier releases to 24.10 retain the ix-applications dataset.
+During the migration process, 24.10 reads the stored Kubernetes app data in the ix-applications dataset, ports them to Docker, and saves them in the new ix-apps dataset.
+App storage ixVolumes present in ix-applications are cloned under the ix-apps dataset and promoted.
+
+The app data retained in ix-applications enables TrueNAS to revert to 24.04 with functional applications.
+TrueNAS 24.10 or newer does not use app data in the ix-applications dataset.
+It can be safely removed after fully migrating to 24.10, but apps will not function if reverted to 24.04 without the ix-applications dataset.
+{{< /hint >}}
+
+**ixVolume Datasets**
+
+**ixVolume** datasets allow TrueNAS to automatically create an app storage path inside the hidden ix-apps dataset.
+
+ixVolumes are typically created with appropriate permissions to deploy the application.
+If needed, use **Enable ACL** in **Storage Configuration** to configure ACL entries for an ixVolume.
+
+ixVolumes are not recommended for permanent storage volumes, they are intended for use as rapid storage for a test deployment of the container.
+Though they can simplify test deployment, ixVolumes complicate tasks like app data backup.
+We recommend manually adding datasets and configuring container storage volumes with the host path option.
+
+**Host Path Datasets**
+
+**Host Paths** allow users to mount existing TrueNAS datasets to paths within the app container.
+Create the TrueNAS dataset(s) before assigning them as host paths within the app installation screen.
+
+Mounting a host path does not automatically configure appropriate permissions to deploy the application.
+Use **Enable ACL** in **Storage Configuration** to configure ACL entries for each host path.
+
+{{< expand "Configuring Host Path ACL Entries" "v" >}}
+After entering the path inside the container in **Mount Path**, select **Enable ACL**.
+Browse to or enter the path to the dataset in **Host Path**.
+Click **Add** next to **ACL Entries** to display a set of ACE fields.
+Use **ID Type** to select whether the ACE is for a user or a group.
+Enter the UID or GID in **ID** and adjust the permissions level in **Access**.
+
+Refer to the app **Run As Context** on the app details screen for default ID requirements.
+A user or group ID does not need to exist locally on TrueNAS or match the name configured in the container to grant an ACE.
+Failing to configure host path ACLs prevents the app from deploying!
+
```

Comment:
suggest adding this content to the existing snippet that covers this. I think I created a new snippet with this in the Plex PR.

Reply:
I don't see anything in that PR. I have added this both here and in AppsInstallWizardSettings.md